docs/contributing/index.md

```diff
@@ -17,7 +17,7 @@ To set up your environment for success, follow the [technical setup](technical-s
 
 ## Using this repo
 
-Review existing [Starport issues](../../issues/) to see if your question has already been asked and answered.
+Review existing [Starport issues](https://github.com/tendermint/starport/issues) to see if your question has already been asked and answered.
```

Comment:
@loross14 to follow semantic PR guidelines, you have to add `docs:` in start of commit name so GA can pass

Reply:
all set!